### PR TITLE
feat(UtilityScript): Script for couchdb 2.x to update a field(String) in project document to a new value.

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Script for couchdb 2.x to update a field(String) in project document to a new value.
> * Which issue is this pull request belonging to and how is it solving it? (*#1055*)
> * Did you add or update any new dependencies that are required for your change?

      Following variables needs to be updated based on the requirement
      1. `fieldName` = *"field_to_be_migrated"*
      2. `oldValue` = *"old_value"*
      3. `newValue` = *"new_value"*

> Have you implemented any additional tests? - No

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

**Review by mcjaeger — APPROVED** [review]
test works, code review not done

File: scripts/utilities/003_update_project_field_value_couchdb_2_x.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2019. Part of the SW360 Portal Project.
```
Comment (smrutis1) [inline scripts/utilities/003_update_project_field_value_couchdb_2_x.py:3]: May be the Copyright year should be updated.

Comment (JaideepPalit) [reply scripts/utilities/003_update_project_field_value_couchdb_2_x.py]: done

**Comment by mcjaeger** [comment]
code looks good.

**Pull request merged**